pkgs/applications/editors/inventic/skipper.nix

```diff
@@ -0,0 +1,65 @@
+{ stdenv, lib, fetchurl, autoPatchelfHook, makeWrapper, unzip,
+
+# Unpack libs
+boost172, curl, libX11, libpng12, libxml2, libxslt, openssl, postgresql, qt5
+, zlib }:
+
+with lib;
+
+stdenv.mkDerivation rec {
+  pname = "skipper";
+  version = "3.2.25.1701";
+  src = fetchurl {
+    url =
+      "https://downloads.skipper18.com/${version}/Skipper-${version}-Linux-all-64bit.zip";
+    sha256 = "130wfn20mj0nrp66grk7fj8xr6m4l1cp7741w40ixp8gdhih57zc";
+  };
+
+  nativeBuildInputs = [ unzip autoPatchelfHook qt5.wrapQtAppsHook makeWrapper ];
+
+  buildInputs = [
+    boost172
+    curl
+    libX11
+    libxml2
+    libxslt
+    openssl
+    postgresql
+    qt5.qtbase
+    qt5.qtscript
+    stdenv.cc.cc.lib
+  ];
+
+  runtimeDependencies = [ libpng12 ];
+
+  unpackPhase = ''
+    unzip $src
+  '';
+
+  installPhase = ''
+    mkdir -p $out/lib
+    mv ./libs/libmysqlclient.so* $out/lib
+    mv ./libs/libQtitan*.so* $out/lib
+    rm -r ./libs ./bearer ./platforms
+
+    mkdir -p $out/opt
+    chmod +x ./Skipper
+    mv ./* $out/opt
+
+    makeWrapper $out/opt/Skipper $out/bin/skipper \
+      --prefix LD_LIBRARY_PATH : ${makeLibraryPath runtimeDependencies}
+  '';
+
+  postFixup = ''
+    wrapQtApp $out/opt/Skipper
```

Comment:
Please avoid double wrapping.

Reply:
I tried to remote it, but when I start the package, it fails to launch with the following error:

```
##WARNING: Library libxcb-xinerama.so.0 is missing on your system. Please install it by using 'sudo apt-get install libxcb-xinerama0'
qt.qpa.plugin: Could not find the Qt platform plugin "xcb" in ""
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

[1]    28569 abort (core dumped)  skipper
```

With the postFixup, everything works well. Do you know why?

Reply:
wrapQtApp uses internally makeWrapper and we can pass the arguments it uses to the same wrapper that already sets LD_LIBRARY_PATH.